src/js/components/MaskedInput/StyledMaskedInput.js

```diff
@@ -39,6 +39,11 @@ export const StyledMaskedInput = styled.input`
     outline: none;
   }
 
+  &:-moz-placeholder, // FF 18-
+  &::-moz-placeholder { // FF 19+
+    opacity: 1;
+  }
```

Comment:
Code duplication with StyledTextInput, we should extract it to utils.

Reply:
@ShimiSun I've extract the duplication to utils and added it to `inputStyle`. The snapshot changes are not truly changes but rather just the css being a different order from what it previously was.